supportedBackends/htcondor/src/main/scala/cromwell/backend/impl/htcondor/HtCondorJobExecutionActor.scala

```diff
@@ -136,9 +152,23 @@ class HtCondorJobExecutionActor(override val jobDescriptor: BackendJobDescriptor
   /**
     * Abort a running job.
     */
-  override def abort(): Unit = throw new UnsupportedOperationException("HtCondorBackend currently doesn't support aborting jobs.")
+  override def abort(): Unit = {
+    // Nothing to do in case `condorJobId` is not defined
+    condorJobId foreach { id =>
+      log.info("{}: Aborting job [{}:{}].", tag, jobDescriptor.key.tag, id)
+      val abortProcess = new HtCondorProcess
+      val argv = Seq(HtCondorCommands.Remove, id)
+      val process = abortProcess.externalProcess(argv)
+      val exitVal = process.exitValue()
```

Comment:
Ideally the process which is killing the task wouldn't be synchronous inside the receive block